libraries/botbuilder-applicationinsights/src/applicationInsightsTelemetryClient.ts

```diff
@@ -94,33 +97,56 @@ export class ApplicationInsightsTelemetryClient implements BotTelemetryClient, B
         return this.config;
     }
 
-    /* defaultClient()
+    /**
      * Provides direct access to the telemetry client object, which might be necessary for some operations.
      */
     get defaultClient(): appInsights.TelemetryClient {
         return this.client;
     }
 
+    /**
+     * Sends information about an external dependency (outgoing call) in the application.
+     * @param telemetry Telemetry Dependency.
+     */
     public trackDependency(telemetry: TelemetryDependency): void {
         this.defaultClient.trackDependency(telemetry as appInsights.Contracts.DependencyTelemetry);
     }
 
+    /**
+     * Logs custom events with extensible named fields.
+     * @param telemetry Telemetry Event.
+     */
     public trackEvent(telemetry: TelemetryEvent): void {
         this.defaultClient.trackEvent(telemetry as appInsights.Contracts.EventTelemetry);
     }
 
+    /**
+     * Logs a system exception.
+     * @param telemetry Telemetry Exception.
+     */
     public trackException(telemetry: TelemetryException): void {
         this.defaultClient.trackException(telemetry as appInsights.Contracts.ExceptionTelemetry);
     }
 
+    /**
+     * Sends a trace message.
+     * @param telemetry Telemetry Trace.
+     */
     public trackTrace(telemetry: TelemetryTrace): void {
         this.defaultClient.trackTrace(telemetry as appInsights.Contracts.TraceTelemetry);
     }
 
+    /**
+     * Logs a dialog entry / as an Application Insights page view.
```

Comment:
Is this char ok? It seems to be but I dont know.
```suggestion
     * Logs a dialog entry as an Application Insights page view.
```

Reply:
I think it was like this in .NET, but you are right. Fixed. Thanks!!